sklearn/ensemble/_stacking.py

```diff
@@ -277,6 +297,9 @@ class StackingClassifier(ClassifierMixin, _BaseStacking):
         Refer :ref:`User Guide <cross_validation>` for the various
         cross-validation strategies that can be used here.
 
+        If “prefit” is passed, it is assumed that all base_estimators have
+        been fitted already.
```

Comment:
Please note the risk of overfitting if the models have been trained on the same data that will now be used to train the stacking model.